dump_manager_bmc.cpp

```diff
@@ -166,7 +166,20 @@ void Manager::checkAndCreateCoreDump()
             log<level::INFO>(
                 fmt::format("Core file found, files size {}", files.size())
                     .c_str());
-            captureDump(Type::ApplicationCored, files);
+
+            try
+            {
+                captureDump(Type::ApplicationCored, files);
+            }
+            catch (const std::exception& excp)
```

Comment:
Can we have derived class exception first followed by standard one? If possible then that would be more robust and verbose.